src/core/lib/surface/completion_queue.h

```diff
@@ -47,6 +47,16 @@ typedef struct grpc_cq_completion {
   uintptr_t next;
 } grpc_cq_completion;
 
+/// For callback CQs, the following is what is actually intended by
```

Comment:
Ok, after reading `completion_queue.cc` I understood what the comment meant i.e that the `tags` queued in the Callback completion queues **must**  point to objects of type `CqCallbackInterface`. 
It would help to make the comment clearly state this.

Reply:
I'll fix this comment.